### PR TITLE
SPCTR-3183 [Core] Adding plugin update message hook

### PR DESCRIPTION
**Pull request by adi3890**
### Description
Adding hook `in_plugin_update_message-' . UAGB_BASE`
This will be helpful to send any further Upgrade Notice for any major release

### Screenshots
image

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: classes/class-uagb-update.php [hunk 1]
```diff
@@ -157,6 +159,52 @@ public function fresh_install_update_asset_generation_option() {
...
+		/**
+		 * Plugin update notification.
+		 *
+		 * @param array  $data Plugin update data.
+		 * @param object $response Plugin update response.
+		 * @since x.x.x
+		 * @return void
+		 */
+		public function plugin_update_notification( $data, $response ) {
+			if ( ! empty( $data['upgrade_notice'] ) ) { ?>
```
Comment (bsf-zanev) [inline classes/class-uagb-update.php:172]: Should we use `if(): endif;` for easier readablility?

**Review by bsf-zanev — APPROVED** [review]
Looks good to me.

**Review by rahulvarma722 — APPROVED** [review]

**Review by sarangshahane — APPROVED** [review]
The changes looks good to me.

**Review by AbhijitNage123 — APPROVED** [review]

**Review by aapatre — APPROVED** [review]

File: classes/class-uagb-update.php (hunk 1 above)
Comment (aapatre) [reply classes/class-uagb-update.php:172]: I think we can use the php comments ( `//endif`) , let me know your opinion on the same

Comment (adi3890) [reply classes/class-uagb-update.php:172]: adding //endif

**Pull request merged**